process/best_practices_badge.md

```diff
@@ -12,7 +12,73 @@ This document outlines specific questions Academy Software Foundation projects h
 * TOC
 {:toc}
 
-## "The project MUST have at least one primary developer who knows how to design secure software." - how to fulfill this?
+## Passing
+
+### The project website MUST succinctly describe what the software does (what problem does it solve?).
+
+For projects without a website, the primary GitHub repo ```README.md``` should have this information.
```

Comment:
If a project is publicly visible on GitHub, but definition it *HAS* a website: the source code's repo site.

I'd change "without a website" to "without a separate website". Then add, "If a project doesn't have a separate website, but has a public GitHub or GitLab source code repository site, that repository site *is* the project website."